pkg/metric/prometheus/http/metric_server.go

```diff
@@ -0,0 +1,142 @@
+package http
```

Comment:
Do we need an extra package for this? Maybe move it up into the prometheus package and name the factory `NewMetricsServer`? :thinking:

Reply:
moved it up to prometheus, it was causing import cycle because the metric_server is using the kernel pkg